src/Summoner/GhcVer.hs

```diff
@@ -0,0 +1,49 @@
+module Summoner.GhcVer
+       ( GhcVer (..)
+       , showGhcVer
+       , parseGhcVer
+       , latestLts
+       , baseVer
+       ) where
+
+import Relude
+import Relude.Extra.Enum (inverseMap)
+
+-- | Represents some selected set of GHC versions.
+data GhcVer
+    = Ghc7103
+    | Ghc801
+    | Ghc802
+    | Ghc822
+    | Ghc843
```

Comment:
Should we already add 8.6.1? Or it's too soon?
It's okay with cabal, but stack will bring a lot of issues with it :(

Reply:
Yeah, we don't have logic that allows to add `GhcVer` only to `cabal`. So I guess we need to wait for it...